roles/openshift_health_checker/test/openshift_check_test.py

```diff
@@ -81,6 +81,7 @@ class TestCheck(OpenShiftCheck):
 @pytest.mark.parametrize("keys,expected", [
     (("foo",), 42),
     (("bar", "baz"), "openshift"),
+    (("bar.baz",), "openshift"),
```

Comment:
IIRC we had this before and then removed the "extra feature" in favor of a single way of doing it.
Are you intending to use both `get_var('openshift.foo.bar')` and `get_var('openshift', 'foo', 'bar')`?